packages/jwt-verifier/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/jwt-verifier",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
does this need a version bump (or even a changelog entry)? As far as I can tell this change does not affect the published module, but only developers who clone the repo.

Reply:
ok, will remove it. before I saw we started adding `others` in the changelog, I thought we want to expose every update to the repo ...

Reply:
Most updates (fix/feature/other) will go in the CHANGELOG, but "chore" tasks like updating lint or fixing flakey tests, I don't think needs any doc or release because it does not affect the published package.